.copywrite.hcl

```diff
@@ -0,0 +1,9 @@
+project {
+  license = "MPL-2.0"
+  copyright_year = 2013
```

Comment:
```suggestion
  copyright_year = 2023
```

Reply:
Ah I see where this change came from. Good catch. This year should actually match the year in the LICENSE file which is 2021. I'll update this and merge